daemon/src/payout_curve.rs

```diff
@@ -101,8 +103,10 @@ fn calculate_payout_parameters(
             let right_bound = row[1] as u64;
             let long_amount_btc = row[2];
 
+            // TODO: cover the case where the maker pays funding fees
+
             let long_amount = to_sats(long_amount_btc)?;
-            let long_amount_adjusted = long_amount.saturating_sub(funding_fee);
+            let long_amount_adjusted = long_amount.saturating_sub(funding_fee.as_u64());
```

Comment:
How about we convert `FundingFee` into `bitcoin::Amount` instead and do the math before we call `to_sats`?

Reply:
Perhaps this could be done later if needed? this is an improvement that can be done at any stage - it was `u64` already. (note: I will need to revisit this anyway when allowing charging the maker)